### ISSUE TITLE
Switching to design execution environment when opening a project.

### ISSUE THREAD

**Issue by vitvakatu** [enso-org/enso#5932]
When the project is opened and the execution environment is set to `live`, we should switch it to `design`. As there are only two execution environments now, we can just set `design` unconditionally each time we open the project.

Switching the environment should be done before creating the execution context.

### QA Acceptance Criteria

```
GIVEN a project with execution mode set to `production` in package.yml
WHEN The project is opened in the IDE
THEN I see that the current execution mode is set to `development`
```

**Comment by vitvakatu** [comment]
No longer needed, according to the latest design.
